test/fenced_code_without_blank_lines.md

```diff
@@ -39,4 +39,4 @@ text
 
 ```
 code at end of file without newline
-```
\ No newline at end of file
+``` {MD047:42}
```

Comment:
Sorry, I wasn't clear. This syntax lets you move the {} part to a different line of the file. In this case, somewhere above in the file so it's not on the last line.

Reply:
I was wondering how I am affecting the test on this file. Is this just about better readability, or is it in fact affecting test scenario? I like readability and don't want to unintentionally affect some test, so I am happy to adjust as needed. Just want to make sure I know what I am doing this time :)

Is this change what you are looking for?

````Markdown
```
code at end of file without newline {MD047:42}
```
````

Reply:
Yes, that’s what I am suggesting. Probably it doesn’t matter, but this file is specifically to test different code fence scenarios and ending one immediately after the close seems like a good test case that I want as part of the suite. :)